frontend/packages/console-shared/src/components/formik-fields/SwitchField.tsx

```diff
@@ -1,40 +1,15 @@
 import * as React from 'react';
-import { useField } from 'formik';
-import { FormGroup, Switch } from '@patternfly/react-core';
-import { getFieldId } from './field-utils';
+import { Switch } from '@patternfly/react-core';
 import { CheckboxFieldProps } from './field-types';
+import ToggleableFieldBase from './ToggleableFieldBase';
 
-const SwitchField: React.FC<CheckboxFieldProps> = ({
-  label,
-  formLabel,
-  helpText,
-  required,
-  ...props
-}) => {
-  const [field, { touched, error }] = useField(props.name);
-  const fieldId = getFieldId(props.name, 'checkbox');
-  const isValid = !(touched && error);
-  const errorMessage = !isValid ? error : '';
-  return (
-    <FormGroup
-      fieldId={fieldId}
-      label={formLabel}
-      helperText={helpText}
-      helperTextInvalid={errorMessage}
-      isValid={isValid}
-      isRequired={required}
-    >
-      <Switch
-        {...field}
-        {...props}
-        id={fieldId}
-        label={label}
-        isChecked={field.value}
-        aria-describedby={`${fieldId}-helper`}
-        onChange={(value, event) => field.onChange(event)}
-      />
-    </FormGroup>
-  );
-};
+const SwitchField: React.FC<CheckboxFieldProps> = (baseProps) => (
+  <ToggleableFieldBase {...baseProps}>
+    {(props) => {
+      delete props.isValid;
```

Comment:
Why is this here?

If absolutely necessary it's better to not modify props.

Do this instead.
`({ isValid, ...props }) => {`

Reply:
`isValid` is not used on Switch PF4 component and it passes rest of the props to a DOM element, so it results in `backend.js:6 Warning: React does not recognise the `isValid` prop on a DOM element.` error. 

Fixed.